internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -797,9 +797,18 @@ func convertPublish(topics []manifest.Topic, accountID, region, app, env, svc st
 	var publishers template.PublishOpts
 	// convert the topics to template Topics
 	for _, topic := range topics {
+		var fifo template.Fifo
+		if !topic.Fifo.IsEmpty() {
+			if aws.BoolValue(topic.Fifo.Enable) {
+				fifo.Enable = aws.BoolValue(topic.Fifo.Enable)
+			}
+			if !topic.Fifo.Advanced.IsEmpty() {
+				fifo.ContentBasedDeduplication = aws.BoolValue(topic.Fifo.Advanced.ContentBasedDeduplication)
```

Comment:
What do you think of adding more methods to the manifest similar to this feedback: https://github.com/aws/copilot-cli/pull/4017#discussion_r976809646

For example, a sample API might be:
```go
// FIFO returns a first-in first-out SNS topic and true if the underlying topic representation has FIFO enabled.
func (t Topic) FIFO() (FIFOTopic, bool)

type FIFOTopic Topic
func (t FIFOTopic) HasContentBasedDeduplication() bool
```